extensions/2.0/Khronos/KHR_materials_unlit/README.md

```diff
@@ -0,0 +1,143 @@
+`KHR_materials_unlit` extension to any glTF material. When present, the
+extension indicates that a material should be unlit and use available
+`baseColor` values, alpha values, and vertex colors while ignoring all
+properties of the default PBR model related to lighting or color. Alpha
+coverage and doubleSided still apply to unlit materials.
+
+```json
+{
+    "materials": [
+        {
+            "name": "MyUnlitMaterial",
+            "pbrMetallicRoughness": {
+                "baseColorFactor": [ 0.5, 0.8, 0.0, 1.0 ]
+            },
+            "extensions": {
+                "KHR_materials_unlit": {}
+            }
+        }
+    ]
+}
+```
+
+### Definition
+
+The Unlit material model describes a constantly shaded surface that is
+independent of lighting. The material is defined only by properties already
+present in the [glTF 2.0 material specification](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#material).
+No new properties are added by this extension — it is effectively a boolean
+flag indicating use of an unlit shading model. Additional properties on the
+extension object are allowed, but may lead to undefined behaviour in conforming
+viewers.
+
+Color is calculated as:
+
+```
+color = <baseColorTerm>
+```
+
+`<baseColorTerm>` is defined by the [core glTF material specification](https://github.com/KhronosGroup/glTF/blob/master/specification/2.0/README.md#metallic-roughness-material).
+
+### Example
+
+The following example defines an unlit material with a constant red color.
+
+```json
+"materials": [
+    {
+        "name": "red_unlit_material",
+        "pbrMetallicRoughness": {
+            "baseColorFactor": [ 1.0, 0.0, 0.0, 1.0 ]
+        },
+        "extensions": {
+            "KHR_materials_unlit": {}
+        }
+    }
+]
+```
+
+### Extension compatibility and fallback materials
+
+When possible, authoring tools should provide a fallback material definition
+and mark the `KHR_materials_unlit` extension as optional. Models
+including the extension optionally will still render in all clients that
+support the core glTF 2.0 specification, falling back to a PBR metal/rough
+material. In clients that do not support the extension, direct lighting will
+be applied and performance may be degraded.
+
+Defining a fallback to the default PBR shading model is achieved by setting
+appropriate parameters on the default material.
+
+```json
+"materials": [
+    {
+        "name": "unlit_with_fallback",
+        "pbrMetallicRoughness": {
+            "baseColorFactor": [ 1.0, 1.0, 1.0, 1.0 ],
+            "baseColorTexture": { "index": 0 },
+            "metallicFactor": 0.0
+        },
+        "extensions": {
+            "KHR_materials_unlit": {}
+        }
+    }
+]
+```
+
+> **Implementation Note:** For best fallback behavior in clients that do not
+> implement the `KHR_materials_unlit` extension, authoring tools may use:
+>
+> * `metallicFactor` and `emissiveFactor` are `0`.
```

Comment:
Maybe is clear enough already but maybe being even more explicit as `emissiveFactor` is a vector?:  `metallicFactor is 0 and emmisiveFactor is [0, 0, 0]`.

Reply:
Done 👍